CoreFoundation/Base.subproj/ForSwiftFoundationOnly.h

```diff
@@ -27,7 +27,9 @@
 #include <CoreFoundation/ForFoundationOnly.h>
 #include <fts.h>
 #include <pthread.h>
+#ifndef __ANDROID__
```

Comment:
This is going to have conflict with https://github.com/apple/swift-corelibs-foundation/pull/1249